src/Sylius/Bundle/ApiBundle/Resources/config/services/validator.xml

```diff
@@ -16,7 +16,7 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
 >
     <services>
-<!--        <defaults public="true" />-->
+        <defaults public="true" />
```

Comment:
Why did you set all services to public? I think it would be better to make only your new service public.